sys/net/gnrc/netif/ethernet/gnrc_netif_ethernet.c

```diff
@@ -34,6 +34,8 @@
 static int _send(gnrc_netif_t *netif, gnrc_pktsnip_t *pkt);
 static gnrc_pktsnip_t *_recv(gnrc_netif_t *netif);
 
+static char addr_str[ETHERNET_ADDR_LEN * 3];
```

Comment:
unused if `ENABLE_DEBUG` is not set.

Reply:
I thought it would generate a warning about the unused variable, but I did not test this.

Reply:
Elsewhere in the code `IPV6_ADDR_MAX_STR_LEN` is used instead.